charmcraft/commands/store/__init__.py

```diff
@@ -18,8 +18,11 @@
 
 import logging
 
+from tabulate import tabulate
```

Comment:
Shouldn't this be grouped with logging?

Reply:
Nop, logging is stdlib, tabulate is an external module.